base_rvest_web_scapping.Rmd

```diff
@@ -0,0 +1,135 @@
+
+## 2.Understanding a web page "HTML"
+
+HTML is not a programming language. Instead, it’s called a markup language — it describes the content and structure of a web page. HTML is organized using tags, which are surrounded by <> symbols. Different tags perform different functions. Together, many tags will form and contain the content of a web page.
+
+The simplest HTML document looks like this:
+
+```{}
+<html>
+<head>
+```
+
+Although the above is a legitimate HTML document, it has no text or other content. If we were to save that as a html file and open it using a web browser, we would see a blank page.
+
+Notice that the word html is surrounded by <> brackets, which indicates that it is a tag. To add some more structure and text to this HTML document, we could add the following:
+
+```{}
+<head>
+</head>
+<body>
+<p>
+Here's a paragraph of text!
+</p>
+<p>
+Here's a second paragraph of text!
+</p>
+</body>
+</html>
+```
+
+Here we’ve added \<head> and \<body> tags, which add more structure to the document. The \<p> tags are what we use in HTML to designate paragraph text.
+
+There are many, many tags in HTML. The important takeaway is to know that tags have particular names (html, body, p, etc.) to make them identifiable in an HTML document.
+
+Notice that each of the tags are “paired” in a sense that each one is accompanied by another with a similar name. That is to say, the opening \<html> tag is paired with another tag \</html> that indicates the beginning and end of the HTML document. The same applies to \<body> and \<p>.
+
+
+## 3.The rvest library
+
+The rvest library is a library that lets users easily scrape data from web pages. Rvest is one of the tidyverse libraries, so it works well with the other libraries contained in the bundle. 
+
+Install package
+```{r}
+#install.packages('rvest')
+library(rvest)
+```
+
+In order to start parsing through a web page, we first need to request that data from the computer server that contains it. In rvest, the function that serves this purpose is the read_html() function. 
+We will get data from "https://www.beckershospitalreview.com/public-health/states-ranked-by-percentage-of-covid-19-vaccines-administered.html", which is in the problem2.
+
+read_html():
+```{r}
+url <- "https://www.beckershospitalreview.com/public-health/states-ranked-by-percentage-of-covid-19-vaccines-administered.html"
+page <- read_html(url)
+```
+
+To read the webpage only once, we need to save the page and read the saved page next time when we try to run the code. We can use function write_html().
+
+write_html():
+```{r}
+write_html(page, "vac_page.html")

```

Comment:
The file should not contain any write functions. Please take https://jtr13.github.io/cc21/tutorial-for-pull-request-mergers.html#check-.rmd-file-contents  15.5 as a reference